test/cli/metrics-file/metrics-file.out

```diff
@@ -16,6 +16,8 @@ No errors! Great job.
    "value": 1
    "name": "ruby_typer.unknown..types.input.classes.total",
    "value": 5
+   "name": "ruby_typer.unknown..singleton_classes",
```

Comment:
Since this metrics is now public we can give it a better name if you prefer.

Reply:
Yeah can you name it like the others?

Reply:
Went with `types.input.singleton_classes.total` 👍